js_modules/dagit/packages/core/src/schedules/ScheduleMutations.tsx

```diff
@@ -4,11 +4,14 @@ import * as React from 'react';
 import {showCustomAlert} from '../app/CustomAlertProvider';
 import {PythonErrorInfo, PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorInfo';
 
-import {StartSchedule, StartSchedule_startSchedule_PythonError} from './types/StartSchedule';
+import {
+  StartThisSchedule,
+  StartThisSchedule_startSchedule_PythonError,
+} from './types/StartThisSchedule';
 import {StopSchedule, StopSchedule_stopRunningSchedule_PythonError} from './types/StopSchedule';
 
 export const START_SCHEDULE_MUTATION = gql`
-  mutation StartSchedule($scheduleSelector: ScheduleSelector!) {
+  mutation StartThisSchedule($scheduleSelector: ScheduleSelector!) {
```

Comment:
`StartScheduleMutation` is a generated schema type, and is also the name of the type generated for this specific mutation, which leads to a TypeScript error due to multiple types having the same name. To sidestep this, I just renamed the mutation.